src/ts/entrypoint.js

```diff
@@ -0,0 +1,19 @@
+import "../../tmp/js/constants.js";
```

Comment:
a. should this be .ts?
b. Could these just be imports of the ts files without a path?